### PR TITLE
Interop 656 Due to incorrect owner of dmapi-error file tvault-datamover-api service is failed to start

### PR DESCRIPTION
**Pull request by trilio-shiwanand**

### CODE REVIEW

File: ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml [hunk 1]
```diff
@@ -62,6 +62,42 @@
...
+  when:
+    - dmapi_error_log.stat.exists == true
+
+- name: Enable tvault-datamover-api service
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+  with_items:
+    - tvault-datamover-api
```
Comment (shyam-biradar) [inline ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml:102]: Haven't we updated service name to 'triliovault-datamover-api'?
Please open a bug on dev side.

File: ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml [hunk 2]
```diff
@@ -62,6 +62,42 @@
...
+  notify:
+      - restart tvault-datamover-api
+
 - name: ensure /tmp/datamover_url file is deleted
   file: path="/tmp/datamover_url" state=absent
 
+- name: Disable tvault-datamover-api service
+  service:
+    name: "{{ item }}"
+    state: stopped
```
Comment (shyam-biradar) [inline ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml:77]: We should not need to stop this service. We should set correct permissions on log directory before starting the dmapi service. Please add changes.

**Review by shyam-biradar — CHANGES_REQUESTED** [review]

File: ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml (hunk 1 above)
Comment (trilio-shiwanand) [reply ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml:102]: Sure will open JIRA

File: ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml (hunk 2 above)
Comment (trilio-shiwanand) [reply ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml:77]: sure

**Pull request closed without merging**